Marketplace/Providers/provider-onboarding-checklist.md

```diff
@@ -0,0 +1,50 @@
+{{{
+"title": "Marketplace Onboarding Checklist",
+"date": "05-05-2017",
+"author": "Brandy Smith",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+The CenturyLink Marketplace allows customers to purchase software from an extensive list of Software Providers that partner with CenturyLink. During the CenturyLink Marketplace Onboarding Program, there are a variety of tasks that not only need to be completed by the CenturyLink Marketplace team, but the provider as well. The Onboarding Program is meant to streamline the process of getting partner products into the Marketplace. This Knowledge Base Article provides a Checklist for Providers to follow throughout the onboarding process.
+
+:white_check_mark:	**Create CenturyLink Cloud Account (Responsible Party: Provider)**
+
+* The account can be created from [here](https://www.ctl.io/free-trial/).
+
+* There is no cost associated with creating or maintaining your account.
+
+* You will immediately receive a $500 credit.
+
+* Additionally, as part of the Ecosystem program, you will receive an additional $2,000/month stipend for the creation & maintenance of your solution on CenturyLink Cloud.
+
+* You will be emailed your username & password for CenturyLink Cloud. You must login into the CenturyLink Cloud Portal to change your password.
+
+:white_check_mark: **Provider Portal Access (Responsible Party: Provider)**
+
+* Once you have created your CenturyLink Cloud account, you will also have access to the Provider Portal (https://provider-portal.ctl.io).
+
+* There is training material located within the provider portal.
+
+* There is additional training material available in the [Getting Started Guide: Marketplace & Provider Portal](../Marketplace/Providers/getting-started-guide-marketplace-ecosystem-provider-portal.md), which will serve as your primary guide for creating your product in the Provider Portal. We recommend using it in conjunction with the training videos provided within the Portal.
```

Comment:
Change link to `./getting-started-guide-marketplace-ecosystem-provider-portal.md`